src/server/cluster/slot_set.h

```diff
@@ -0,0 +1,101 @@
+// Copyright 2024, DragonflyDB authors.  All rights reserved.
+// See LICENSE for licensing terms.
+//
+
+#pragma once
+
+#include <bitset>
+#include <memory>
+#include <vector>
+
+namespace dfly {
+
+using SlotId = uint16_t;
+
+struct SlotRange {
+  SlotId start = 0;
+  SlotId end = 0;
+};
+
+using SlotRanges = std::vector<SlotRange>;
+
+class SlotSet {
+ public:
+  static constexpr SlotId kMaxSlot = 0x3FFF;
```

Comment:
nit: maybe it's better to use the number that redis uses in decimal format? 16k something... at least it's recognizable visually

Reply:
I prefer hex format)